src/bugsnag.js

```diff
@@ -708,6 +711,29 @@
     });
   }
 
+  // creates a function that takes a timeStamp and returns the number of milliseconds it happened in
+  // the past.
+  //
+  // This is necessary because depending on the browser the event.timeStamp could be a
+  // DOMTimeStamp or a DOMHighResTimeStamp
+  function makeMillisecondsAgo() {
+    function timeNear(a, b) {
+      var d = 1000 * 60 * 5;
+      return a > b - d && a < b + d;
+    }
+
+    var testTimeStamp = (document.createEvent("CustomEvent")).timeStamp;
```

Comment:
You are correct. That's my fault. #146 should fix it though. How does that look to you?